roles/static_inventory/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+# Either to checkpoint the dynamic inventory into a static one
+refresh_inventory: True
+inventory: static
+inventory_dir: ~/openstack-inventory
```

Comment:
I'm not sure, that would work as expected with the follow up [PR](https://github.com/openshift/openshift-ansible-contrib/pull/550/files?short_path=9141c3c#diff-9141c3c37ff1e871d38171e178e46e72) . We need this var to control the inventory file placement